ebu_tt_live/xsd1.1/ebutt_datatypes.xsd

```diff
@@ -27,7 +27,7 @@ Please note that the EBU-TT XML Schema is a helping document and NOT normative b
 	<xs:simpleType name="fontSizeType">
 		<xs:restriction base="xs:token">
 			<xs:pattern
-				value="(\d*\.?\d+(px|c|%))|([+-]?\d*\.?\d+(px|c|%))\s([+-]?\d*\.?\d+(px|c|%))"
+				value="([+-]?\d*\.?\d+(px|c|%))(\s([+-]?\d*\.?\d+(px|c|%)))?"
```

Comment:
Okay so I may have spotted an incoherence between 3350 and 3370, in 3370 the fontSizeType is defined like this :
`<ebuttdt:fontSizeType>
: <ebuttdt:lengthType> <whiteSpace> <ebuttdt:lengthType>?`

With :
`<ebuttdt:lengthType>
: <scalar> | <percentage>

<scalar>
: <number> <units>

<number>
: <sign>? <non-negative-number>`

From that syntax definition, a negative number in fontSize is allowed, I will correct that